deploy/manifests/quay-operator/0.0.1/quay-operator.clusterserviceversion.yaml

```diff
@@ -118,21 +108,17 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
-                # NOTE: This controls the default value for `desiredVersion` if left blank.
-                - name: DEFAULT_STREAM
-                  value: downstream
-                image: quay.io/projectquay/quay-operator@sha256:4f88b7231dcac284893e7c6371ee631fccf592b553d0df76e624068bc9bb7553
+                - name: COMPONENT_IMAGE_QUAY
```

Comment:
`RELATED_IMAGE_COMPONENT_QUAY` - prefix `RELATED_IMAGE_` important for OSBS automatic sha substitution